cmd/root.go

```diff
@@ -47,6 +47,7 @@ func init() {
 	flags.Bool("noauth", false, "use the noauth auther when using quick setup")
 	flags.String("username", "admin", "username for the first user when using quick config")
 	flags.String("password", "", "hashed password for the first user when using quick config (default \"admin\")")
+	flags.String("salt", "", "The salt to use when for hashing share password. Can be any value. If changed, existing password-protected shares wil stop working.") //nolint:lll
```

Comment:
Why this salt is needed? According to the https://en.wikipedia.org/wiki/Bcrypt#Description , the salt is automatically included in the output string which means there is no need to add it by yourself. 

```suggestion
	flags.String("salt", "", "The salt is used for hashing share passwords. It can be of any value. If changed, existing password-protected shares will stop working") //nolint:lll
```

Reply:
Removed it